app/exports/isic_export.rb

```diff
@@ -49,8 +49,9 @@ def submit(member, card)
       sendToHome: member.isic_mail_card ? "1" : "0",
       promotionCode: "",
       Optin: member.isic_newsletter ? "1" : "0",
-      postOptOut: "1",
-      postOptOutThird: "1",
+      optinThird: "1",
+      # postOptOut: "1",
+      # postOptOutThird: "1",
```

Comment:
Ah, I edited these because the ISIC staging API differs from the production API. If I didn't send the `optinThird` to the staging API, I'd get an error. I mailed ISIC about the differences and asked them if they could fix the staging API so we can test it decently.